### DISCUSSION TITLE
How can i get pretrained model Cascade R cnn + swin transformer?

### DISCUSSION THREAD

**Discussion by hoya-cho** [open-mmlab/mmdetection#9275] [Q&A] [answered]
How can i get pretrained model Cascade R cnn + swin transformer?

Mostly, there is only the cascade mask rcnn pre-training model, and the cascade rcnn + swin transformer pre-training model cannot be found. Where can I get it?

So, instead of training the cascade mask rcnn pre-learning model with cascade rcnn+ swin transformer, it seems that additional training is not possible because the weight parameters are different, right?

**Comment by jbwang1997** [comment, marked as answer]
You can find Cascade RCNN + Swin in [Swin Transformer official implementation repo](https://github.com/SwinTransformer/Swin-Transformer-Object-Detection)
